test/gtest/conv3d_test_case.hpp

```diff
@@ -36,42 +36,41 @@
 
 struct Conv3DTestCase
 {
+    struct Size3D
```

Comment:
Perhaps we should factor these `SizeND` classes out as common code because we also have them in `group_conv.hpp`.